.github/workflows/scripts/functions/package.json

```diff
@@ -15,7 +15,7 @@
   "main": "lib/index.js",
   "dependencies": {
     "firebase-admin": "^11.5.0",
```

Comment:
We can update to latest version of firebase-admin as long as we are modifying this.